jgrapht-core/src/main/java/org/jgrapht/GraphTests.java

```diff
@@ -260,7 +258,11 @@
     public static <V, E> boolean isStronglyConnected(Graph<V, E> graph)
     {
         Objects.requireNonNull(graph, GRAPH_CANNOT_BE_NULL);
-        return new KosarajuStrongConnectivityInspector<>(graph).isStronglyConnected();
+        if (graph.getType().isUndirected()) {
```

Comment:
The javadoc of this method requires updating too: directed graph -> graph. Moreover, I would specify explicitly that this method supports both directed and undirected graphs. 

Finally, `AbstractStrongConnectivityInspector` probably needs a requiredDirectedGraph instead of objectsNonNull

Reply:
Improved the docs. The `AbstractStrongConnectivityInspector` change was already there.